app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -42,7 +42,7 @@ echo "#"
 if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
     CONSUMER_RPM=$(mktemp --suffix .rpm)
 
-    curl --insecure --output $CONSUMER_RPM <%= subscription_manager_configuration_url %>
+    curl <%= '--insecure' if @insecure %> --output $CONSUMER_RPM <%= subscription_manager_configuration_url %>
```

Comment:
Looks like this always uses HTTP:

https://github.com/Katello/katello/blob/8a1c4fa46d9fb4b0ee941b08f0278a84600e87d6/app/helpers/katello/katello_urls_helper.rb#L5-L18

Do we even need `--insecure` then?

Reply:
Tested, it's not needed there.